src/future.ml

```diff
@@ -411,11 +498,13 @@ module Scheduler = struct
     match jobs with
     | [] -> ()
     | first :: others ->
-      Format.eprintf "\nWaiting for the following jobs to finish: %t@."
-        (fun ppf ->
-           Format.fprintf ppf "[@{<id>%d@}]" first.id;
-           List.iter others ~f:(fun job ->
-             Format.fprintf ppf ", [@{<id>%d@}]" job.id));
+      if !Clflags.verbose then begin
```

Comment:
I've changed it to print the full message, but only if the jobs are still running after 0.5s have elapsed.